data/mods/Magiclysm/monsters/Orcs.json

```
@@ -0,0 +1,103 @@
+[
+  {
+    "id": "mon_orc_warrior",
+    "type": "MONSTER",
+    "name": { "str": "Orc Warrior" },
+    "description": "This green brute is a orc warrior.  It stares at you menacingly, sword in hand.",
+    "default_faction": "orcs",
+    "bodytype": "human",
+    "species": [ "ORC" ],
+    "volume": "60 L",
+    "weight": "70 kg",
+    "hp": 92,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "o",
+    "color": "green",
+    "aggression": 100,
```

```suggestion
    "aggression": 75,
```
This would be the same as the ogre (which still makes them almost certain to attack); the Blood Orc seems a reasonable candidate for aggression 100.